src/hotspot/share/runtime/javaThread.cpp

```diff
@@ -2230,8 +2230,8 @@ void JavaThread::pretouch_stack() {
     if (is_in_full_stack(here) && here > end) {
       size_t to_alloc = here - end;
       char* p2 = (char*) alloca(to_alloc);
-      log_trace(os, thread)("Pretouching thread stack from " PTR_FORMAT " to " PTR_FORMAT ".",
-                            p2i(p2), p2i(end));
+      log_trace(os, thread)("Pretouching thread stack for " UINTX_FORMAT ": " RANGEFMT " (" SIZE_FORMAT ").",
+                            (uintx) osthread()->thread_id(), RANGEFMTARGS(p2, to_alloc), to_alloc);
```

Comment:
Looks to me `RANGEFMTARGS` already prints the number of bytes in range, so printing `to_alloc` is excessive?

Reply:
Grr. Shows that you can make an error in a trivial one line change. Thanks for catching.